build.gradle

```diff
@@ -106,6 +107,7 @@ ext {
     springSecurityVersion = project.hasProperty('springSecurityVersion') ? project.springSecurityVersion : '6.0.0-M5'
     springVersion = project.hasProperty('springVersion') ? project.springVersion : '6.0.0-M4'
     springWsVersion = '4.0.0-M2'
+    slf4jVersion = '1.7.36'
```

Comment:
I wonder if existing `testRuntimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl'` is enough.
My point is to keep deps as short as possible and if we really can use some other existing logging tool instead of extra SLF4J, that would be great.